docs/otel/sdk-metrics.md

```diff
@@ -0,0 +1,393 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: SDK Metrics
+--->
+
+# Semantic Conventions for OpenTelemetry SDK metrics
+
+**Status**: [Development][DocumentStatus]
+
+This document describes metrics emitted by the OpenTelemetry SDK components themselves about their internal state.
+
+<!-- Re-generate TOC with `make markdown-toc` -->
+
+<!-- toc -->
+
+- [Span Metrics](#span-metrics)
+  - [Metric: `otel.sdk.span.created_count`](#metric-otelsdkspancreated_count)
```

Comment:
Nit: should this be `started_count`, to match the `StartSpan` API?

Reply:
Based on @jmacd 's [comment](https://github.com/open-telemetry/semantic-conventions/pull/1631#discussion_r1913572664) "started" seems to refer to only sampled spans, while this metric aims to cover both sampled and unsampled spans. I therefore used the term "created", because it is also used in the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span-creation).